runtime/bindings/python/iree/runtime/system_api.py

```diff
@@ -47,7 +47,7 @@
 
 # Mapping from IREE target backends to their corresponding drivers.
 TARGET_BACKEND_TO_DRIVER = {
-    "dylib-llvm-aot": "local-task",
+    "llvm-cpu": "local-task",
```

Comment:
I guess this isn't load bearing (any more)? Seems like I can make this default mapping change independent of other changes.